designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/StatisticsDb.scala

```diff
@@ -0,0 +1,24 @@
+package pl.touk.nussknacker.ui.db.timeseries
+
+import cats.effect.{IO, Resource}
+
+import scala.concurrent.Future
+import scala.language.higherKinds
+
+trait StatisticsDb[F[_]] extends AutoCloseable {
+  def initialize(): Unit
```

Comment:
I think it's DB-specific and we don't have to include it in the interface. There are two many problems with such methods. Like:
1. who can call it? After all, it's public
2. why does it return `Unit`? What about effects?
3. How can I prevent it from being called more than once?